hudi-client/src/main/java/org/apache/hudi/table/action/rollback/MergeOnReadRollbackActionExecutor.java

```diff
@@ -124,7 +124,7 @@ public MergeOnReadRollbackActionExecutor(JavaSparkContext jsc,
         case HoodieTimeline.COMMIT_ACTION:
           LOG.info("Rolling back commit action. There are higher delta commits. So only rolling back this instant");
           partitionRollbackRequests.add(
-              RollbackRequest.createRollbackRequestWithDeleteDataAndLogFilesAction(partitionPath, instantToRollback));
+              RollbackRequest.createRollbackRequestWithDeleteBaseAndLogFilesAction(partitionPath, instantToRollback));
```

Comment:
there is no rollback plan that gets serialized etc.. so this should be fine

Reply:
+1